eqllib/analytics/execution/T1170-koadic-grouped-behavior-mshta.toml

```diff
@@ -0,0 +1,23 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies an unordered combination of events tied to Koadic's Mshta stager."
+id = "8c238e5c-7f40-4958-b046-1e048709c9b8"
+name = "Koadic Grouped Behavior (Mshta Stager)"
```

Comment:
I think we're still punting on tool-specific detections, at least for now. So I'd rather hold this one off for release.

Reply:
I am fine with this leaving off, example will be in blog.